eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -197,6 +200,11 @@ function Upload-Blobs
     Write-Host "DocDir $($DocDir)"
     Write-Host "Final Dest $($DocDest)/$($PkgName)/$($DocVersion)"
 
+    # Use the step to replace master link to release tag link 
+    $tag = GenerateReleaseTag($PkgName, $DocVersion)
```

Comment:
I don't believe this is going to generate the same link in all cases. Consider "c" where the pkgName here is "docs" but it should be empty if we want it to match the release tag that is generated.